Dockerfile

```diff
@@ -71,7 +71,8 @@ ENV NVCC_THREADS=$nvcc_threads
 ENV VLLM_INSTALL_PUNICA_KERNELS=1
 
 ENV CCACHE_DIR=/root/.cache/ccache
-RUN --mount=type=cache,target=/root/.cache/ccache \
+RUN --mount=type=bind,source=.git,target=.git \
```

Comment:
Note that it's required to bind-mount the `.git` dir in order to be able to extract git information during the build process